### ISSUE TITLE
Live register report is broken for me

### ISSUE THREAD

**Issue by jbarczak** [GPUOpen-Tools/RGA#14]
I have a non-AMD Windows 8.1 machine at home, which I do a lot of side projects on.  I have installed the 1.2 release, and on this machine, RGA is not producing a live register report.   I can get it to produce isa, but no reg report.   Here is a sample session.  

```
c:\scratch>ls
PYRAMID_amdcodexl

c:\scratch>"C:\Program Files\GPUOpen\Radeon GPU Analyzer\rga.exe" --version

OpenCL Error: Can't find AMD platform.
Radeon GPU Analyzer Version: 1.2.58
Failed to initialize.

c:\scratch>"C:\Program Files\GPUOpen\Radeon GPU Analyzer\rga.exe" -s vulkan-spv  --isa ".\pyramid.isa" --livereg ".\livereg_report.txt"  -c Baffin  PYRAMID_amdcodexl

Target GPU detected:

Baffin (Graphics IP v8)
        AMD Radeon (TM) E9260
        AMD Radeon Pro 450
        AMD Radeon Pro 455
        AMD Radeon Pro 460
        AMD Radeon Pro 465
        Radeon (TM) Pro V5300X
        Radeon (TM) Pro WX 4100
        Radeon (TM) Pro WX Series
        Radeon RX 550 Series
        Radeon RX 560 Series
        Radeon(TM) RX 460 Graphics

Building for Baffin... succeeded.

c:\scratch>ls
Baffin_frag_pyramid.isa  PYRAMID_amdcodexl  spvLogInfo.txt

c:\scratch>

```

[scratch.zip](https://github.com/GPUOpen-Tools/RGA/files/1271590/scratch.zip)

**Comment by AmitBM** [comment]
Hi jbarczak,

Thanks for reporting this issue. We will look into that.

**Comment by AmitBM** [comment]
Hi jbarczack,
Please let us know if the fixes in [RGA 1.3](https://github.com/GPUOpen-Tools/RGA/releases/tag/1.3.0.0) solved your problem.

**Comment by jbarczak** [comment]
Works now.  Thanks :)
